openedx/core/lib/xblock_fields/inherited_fields.py

```diff
@@ -0,0 +1,239 @@
+"""
+Inherited fields for all XBlocks.
+"""
+from __future__ import absolute_import
+from datetime import datetime
+
+from django.conf import settings
+from pytz import utc
+
+from xblock.fields import Scope, Boolean, String, Float, XBlockMixin, Dict, Integer, List
+from openedx.core.lib.partitions.partitions import UserPartition
+from .fields import Date, Timedelta
+
+
+DEFAULT_START_DATE = datetime(2030, 1, 1, tzinfo=utc)
```

Comment:
JFTR - I backed away from changing this date, as it's hardcoded in several test locations.